lucet-runtime/lucet-runtime-internals/Cargo.toml

```diff
@@ -24,9 +24,12 @@ memoffset = "0.5.2"
 nix = "0.17"
 num-derive = "0.3.0"
 num-traits = "0.2"
+rand = "0.7"
+rand_core = "0.5"
 raw-cpuid = "6.0.0"
 thiserror = "1.0.4"
 tracing = "0.1.12"
+xorshift = "0.1"
```

Comment:
`xorshift doesn't seem to be used anywhere?

Reply:
Thanks for catching this.  I forgot to push one last time.